src/extensions/default/CodeFolding/main.js

```diff
@@ -303,6 +305,8 @@ define(function (require, exports, module) {
         KeyBindingManager.removeBinding(expandKey);
         KeyBindingManager.removeBinding(collapseAllKey);
         KeyBindingManager.removeBinding(expandAllKey);
+        KeyBindingManager.removeBinding(collapseAllKeyMac);
+        KeyBindingManager.removeBinding(expandAllKeyMac);
```

Comment:
What if we're on a Windows machine, where this shortcut might be assigned to another command?